pkgs/build-support/gcc-wrapper/setup-hook-stdinc.sh

```diff
@@ -0,0 +1,38 @@
+export NIX_CC=@out@
```

Comment:
@edolstra This is an alternate setup hook for gcc-wrapper. As I commented over on 51713fb, KDE packages need their includes to be specified with `-I` rather than `-isystem` because the latter changes gcc's path search order for includes. Many of the KDE applications depend on both Qt 4 and Qt 5 (in different components). The CMake build system selects the order of `-I` flags carefully so that each component sees the correct includes, but gcc-wrapper's use of `-isystem` ruins that careful search order. As a result, several KDE packages cannot be compiled with the gcc-wrapper used in the default stdenv.

The meaning of `StdInc` in `wrapGCCStdInc` is "standard include".